Android/app/src/main/java/app/intra/ui/MainActivity.java

```diff
@@ -572,6 +576,10 @@ private void syncDnsStatus() {
     final TextView indicatorText = controlView.findViewById(R.id.indicator);
     indicatorText.setText(status.activationRequested ? R.string.indicator_on : R.string.indicator_off);
 
+    // Hide server change button by default
+    final Button changeServerButton = controlView.findViewById(R.id.change_server_button);
+    changeServerButton.setVisibility(View.INVISIBLE);
```

Comment:
`syncDnsStatus ` will get called when the new connection is set up, right? I was wondering if we needed to explicitly hide it.

Reply:
Yes.  If the `VpnState` changes, `syncDnsStatus` will get called again with the new state.